orchestration/dags/data_gcp_dbt/models/intermediate/contentful/int_contentful__relationships.sql

```diff
@@ -0,0 +1,4 @@
+SELECT
+    *
```

Comment:
```suggestion
    parent,
    child,
    execution_date
```
Pour ajouter le nom des champs dans le modèle intermediate :)